src/react-slidy-slider.jsx

```diff
@@ -67,10 +72,6 @@ export default class ReactSlidySlider extends Component {
     this.slidyInstance && this.slidyInstance.destroy()
   }
 
-  shouldComponentUpdate (nextProps) {
```

Comment:
Are you sure you want to render on every prop changes?

Don't sure why a false was here but proper considerations have been taken to remove this?